### DISCUSSION TITLE
Multiple fixes on the same location looks a bit odd

### DISCUSSION THREAD

**Discussion by simensrostad** [hello-nrfcloud/web#915] [General]
![image](https://github.com/user-attachments/assets/a78a4c4f-e33f-430a-bbb3-6b3c0b8916fe)

Perhaps there is a way to resolve this by neglecting fixes on the same location using accuracy, but only updating the timestamp.

**Comment by coderbyheart** [comment]
The problem is these are most likely not exactly the same locations. For that feature we have a very crude server-side "cluster" feature, however that basically only works with a minimum distance of 1 km.

But I agree, it would be better to use the map engine's clustering feature: https://maplibre.org/maplibre-gl-js/docs/examples/cluster/

**Reply by gregersrygg** [reply]
The client side clustering looks like a good solution.
